packages/back-end/test/integrations/informationSchemas.test.ts

```diff
@@ -451,3 +458,605 @@ describe("formatInformationSchema", () => {
     ]);
   });
 });
+
```

Comment:
This whole test suite is super flaky due to having to call `new Date()` in the test itself. I'd love any pattern suggestions here.